### PR TITLE
Prepare for execution in Python venv

### PR DESCRIPTION
**Pull request by BjoernAtBosch**

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -5,6 +5,17 @@
             "alias": "local",
             "type": "runtime",
             "programs": [
+                {
+                    "id": "install-deps",
+                    "executable": "python3",
+                    "args": [
```
Comment (erikbosch) [inline manifest.json:11]: Should we better align if we want to use `python`or `python3`
(compare with https://github.com/eclipse-velocitas/devenv-devcontainer-setup/pull/89/files)

I guess it does not have any practical difference as I assume python and python3 both exists and points to the same executable in all of our environments, so it is more of a philosophical question

Comment (BjoernAtBosch) [reply manifest.json:11]: Talked to Dominic: We should use `python3`

Comment (BjoernAtBosch) [reply manifest.json:11]: Will adapt that in devenv-devcontainer-setup ...

**Review by erikbosch — APPROVED** [review]
LGTM

**Pull request merged**